assets/vizList.js

```diff
@@ -1,83 +1,202 @@
+/*
+here we have some essential data for the dashboard, kept seperate from the code. 
+this makes both easier to read and maintain.
```

Comment:
@sarahscottr @wilhitem @ablangworthy27 
obvs this is just a piece of the puzzle content wise, but do these notes make sense/are they helpful?